tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3387,6 +3387,7 @@ sdg/ER_PTD_FRHWTR,Average proportion of Freshwater Key Biodiversity Areas covere
 sdg/ER_PTD_MTN,Average proportion of Mountain Key Biodiversity Areas covered by protected areas
 sdg/ER_PTD_TERR,Average proportion of Terrestrial Key Biodiversity Areas covered by protected areas
 sdg/ER_RSK_LST,Red List Index
+sdg/FP_CPI_TOTL_ZG,"Annual inflation, consumer prices;Annual inflation rate as measured by the consumer price index;Percentage change in the cost to the average consumer of acquiring a basket of goods and services that may be fixed or changed at specified intervals, such as yearly;Annual inflation rate consumer price index"

```

Comment:
the first description "Annual inflation, consumer prices", the second description "annual inflation rate as measured by the consumer price index" and the last description "annual inflation rate consumer price index" are all quite similar, are they all needed? Last year when we did an overhaul on the embeddings, the goal was to have one good distinct description per variable, maybe 2 if necessary